gdcdictionary/schemas/sample.yaml

```diff
@@ -485,7 +487,6 @@ properties:
       - Wedge Resection
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   oct_embedded:
```

Comment:
Remove? Should be covered by `preservation_method`?

Reply:
Removed `oct_embedded`